src/app/Fake.netcore/Program.fs

```diff
@@ -156,7 +156,7 @@ let runOrBuild (args : RunArguments) =
       let result =
         match runResult with
         | Runners.RunResult.SuccessRun warnings ->
-          if warnings <> "" then
+          if warnings <> "" && args.VerboseLevel.PrintNormal then
             traceFAKE "%O" warnings
```

Comment:
Actually shouldn't this print into standard-error anyway? So your tool can just ignore stderr?

Reply:
that's a good point, let me try redirection and see what I get

Reply:
Yes you're right of course :D

Reply:
we can roll back this change then, I think.

Reply:
Yes, this basically means it already works are you are not blocked, correct (besides the sort)?

Reply:
Just remove that line change, please